### DISCUSSION TITLE
Custom success and error content-types in chi wrapper mode

### DISCUSSION THREAD

**Discussion by pboguslawski** [swaggest/rest#115] [Q&A] [answered]
Is it currently possible to define custom content-type header value for success and error responses in place of default `application/json; charset=utf-8` for json responses sent from given using given `r` where

```
r := chirouter.NewWrapper(chi.NewRouter())
```

Something similar to https://github.com/swaggest/rest/blob/v0.2.25/_examples/advanced/router.go#L98 avaliable "in `github.com/swaggest/rest/web` mode".

**Comment by vearutop** [comment, marked as answer]
It is possible to do for `SuccessfulResponseContentType`, but there is no control yet for error content type.

```go
// Router mode.
	r := chirouter.NewWrapper(chi.NewRouter())
	r.Method(http.MethodPost, "/foo", nethttp.SuccessfulResponseContentType(fooUsecase(),"foo/bar")))

// Same with web.Service.
	s := web.DefaultService()
	s.Post("/foo", fooUsecase(), nethttp.SuccessfulResponseContentType("foo/bar"))
```
